packages/react-core/src/components/Drawer/DrawerPanelContent.tsx

```diff
@@ -10,18 +10,38 @@ export interface DrawerPanelContentProps extends React.HTMLProps<HTMLDivElement>
   children?: React.ReactNode;
   /* Flag indicating that the drawer panel should have a border. */
   hasBorder?: boolean;
+  /* Default width for drawer panel */
+  width?: 25 | 33 | 50 | 66 | 75 | 100;
+  /* Drawer panel width on large viewports */
+  widthOnLg?: 25 | 33 | 50 | 66 | 75 | 100;
+  /* Drawer panel width on xl viewports */
+  widthOnXl?: 25 | 33 | 50 | 66 | 75 | 100;
+  /* Drawer panel width on 2xl viewports */
+  widthOn2Xl?: 25 | 33 | 50 | 66 | 75 | 100;
 }
 
 export const DrawerPanelContent: React.SFC<DrawerPanelContentProps> = ({
   className = '',
   children,
   hasBorder = false,
+  width,
```

Comment:
just making sure, we don't need default values for these?

Reply:
@jschuler that's correct, without anything passed through (including defaults) the component maintains the current default settings